docker-compose.yml

```diff
@@ -28,6 +28,9 @@ services:
       FIDES__DEV_MODE: "True"
       FIDES__USER__ANALYTICS_OPT_OUT: "True"
       FIDES__SECURITY__ALLOW_CUSTOM_CONNECTOR_FUNCTIONS: "True"
+      FIDES__SECURITY__BASTION_SERVER_HOST: ${FIDES__SECURITY__BASTION_SERVER_HOST-}
+      FIDES__SECURITY__BASTION_SERVER_SSH_USERNAME: ${FIDES__SECURITY__BASTION_SERVER_SSH_USERNAME-}
+      FIDES__SECURITY__BASTION_SERVER_SSH_PKEY: ${FIDES__SECURITY__BASTION_SERVER_SSH_PKEY-}
```

Comment:
I know it's awful long already but would be good to be explicit about `PKEY` — public vs. private has big implications here :)

Reply:
thanks for fixing!